packages/base/src/asset-registries/Icons.js

```diff
@@ -21,6 +21,9 @@ const registerIconLoader = async (collectionName, loader) => {
 const _loadIconCollectionOnce = async collectionName => {
 	if (!iconCollectionPromises.has(collectionName)) {
 		const loadIcons = loaders.get(collectionName);
+		if (typeof loadIcons !== "function") {
```

Comment:
`if (!loaders.has(collectionName))` is much more meaningful and easy to read. Could also be moved before the `get`